### ISSUE TITLE
Bot don't see my messages

### ISSUE THREAD

**Issue by sebastttt** [kolbytn/mindcraft#268]
I'm write in chat, but bot ignore me

**Comment by BarelyAPrincess** [comment]
I have the same issue. I've been trying to find a cause, but I'm not yet familiar with the code.

**Comment by keydevelops** [comment]
Same issue.

**Comment by BarelyAPrincess** [comment]
The issue exists in both Minecraft 1.20.4 and 1.21.1. The best I can come up with for myself is it will only work if I'm using absolutely no mods. When using some Vanilla+ QoL mods it breaks, and I'm wondering if it steams from a bug in the Mineslayer module that parses the chat messages.

**Comment by flyingtruce** [comment]
Disabling the following chat mods worked for me:
- More Chat History by JackFred
- No Chat Reports by Aizistral (https://www.curseforge.com/minecraft/mc-mods/no-chat-reports)

**Comment by JurassikLizard** [comment]
The server environment cannot be modded with mineflayer.
While it seems you guys are hosting a vanilla server and connecting with a modded client, stuff like modded chats can still get through to the server environment, thus causing problems with mineflayer.
My suggestion is to use strictly QoL mods that don't interact at all with the server, including chats, skins, or other things of that nature.
Performance mods or shaders or stuff like that is fine as long as you're still connecting to a vanilla server.

**Comment by sebastttt** [comment]
Yes, it's possible that i have mods in mods folder, i think that's the reason of the problem. Thank you

**Comment by keydevelops** [comment]
It helped. Thanks.

**Comment by sebastttt** [comment]
Removing mods solve this problem

**Comment by sebastttt** [comment]
Problem not solved, I removed all mods, but bot don't react to my messages

**Comment by Shroototem** [comment]
If you have multiple bots then it requires you to use the `/msg` \ `/w` command to whisper to each bot.

**Comment by BarelyAPrincess** [comment]
> If you have multiple bots then it requires you to use the `/msg` \ `/w` command to whisper to each bot.

Yeah, the problem has nothing to do with having multiple bots or not. I've tested in both scenarios and still experience issues.

**Comment by brebathe** [comment]
My bot just ignores me, it doesn't work, my messages dont go through.

**Comment by uukelele** [comment]
Can you provide console logs of what happens when you send a message? Does it show your message anywhere in the console?

Also, perhaps server-side mods may be to blame (if you have any).
